_books/ion-1-1/theme/highlight.js

```diff
@@ -1236,46 +1236,82 @@ hljs.registerLanguage("ini", function () {
 }());
 hljs.registerLanguage("ion", function () {
     "use strict";
+
     return function (n) {
```

Comment:
🗺️ This code started as a copy/paste of the JSON mode. I rewrote this section to:
1. Un-minify all the variable names for readability.
2. Fix some of the matching rules for macro names in different contexts.